.github/workflow_scripts/run_benchmark.sh

```diff
@@ -45,11 +45,17 @@ done
 # Run dashboard if the branch is not master
 if [ $BRANCH_OR_PR_NUMBER != "master" ]
 then
+    echo "Name of all files"
+    ls
     cwd=`pwd`
-    ls data/results/output/openml/ag_eval/pairwise/* | grep .csv > $cwd/agg_csv.txt
+    ls ./evaluate/pairwise/* | grep .csv > $cwd/agg_csv.txt
     cat agg_csv.txt
     filename=`head -1 $cwd/agg_csv.txt`
+    echo "Printing CSVs"
+    cat $filename
+    echo "Printing all dataset CSV"
+    cat ./evaluate/results_ranked_by_dataset_all.csv
```

Comment:
Is this for debugging purpose? If so, we should remove it. The csv file could be giant and make it hard to read our logs

Reply:
Thanks for pointing it out, had missed it, made the changes.